source/Plugins/ExpressionParser/Clang/ClangHost.cpp

```diff
@@ -67,8 +85,10 @@ bool lldb_private::ComputeClangDirectory(FileSpec &lldb_shlib_spec,
     ++rev_it;
   }
 
+  // Posix-style of LLDB detected.
```

Comment:
I don't understand this comment.

Reply:
Yeah looking back this comment isn't very good. I intended it to indicate we aren't in a framework, but rather a more traditional tree with a `bin/ include/ lib/ share/` set of directories. I can update the comment on llvm.org and cherry-pick it into this PR with the rest of these commits.